adoc/admin-security-certificates.adoc

```diff
@@ -344,63 +329,56 @@ openssl rsa -in /path/to/file.pem -out /path/to/file.key
 ----
 ====
 
-* Replacing the {kube} CA certificate:
-+
-[source,bash]
-----
-mkdir -p my-cluster/pki
-cp <CUSTOM_KUBERNETES_CA_CERT_PATH> my-cluster/pki/ca.crt
-cp <CUSTOM_KUBERNETES_CA_KEY_PATH> my-cluster/pki/ca.key
-chmod 644 my-cluster/pki/ca.crt
-chmod 600 my-cluster/pki/ca.key
-----
+=== Replace OIDC Server Certificate
 
-* Replacing the `etcd` CA certificate:
-+
-[source,bash]
-----
-mkdir -p my-cluster/pki/etcd
-cp <CUSTOM_ETCD_CA_CERT_PATH> my-cluster/pki/etcd/ca.crt
-cp <CUSTOM_ETCD_CA_KEY_PATH> my-cluster/pki/etcd/ca.key
-chmod 644 my-cluster/pki/etcd/ca.crt
-chmod 600 my-cluster/pki/etcd/ca.key
-----
+{productname} uses `oidc-dex` and `oidc-gangway` servers to do authentication and authorization.
+Administrators might choose to replace these server's certificates by issuing a trusted CA certificate after cluster deployment.
+This way, the user does not have to add specific certificates to their trusted keychain.
```

Comment:
Just revisited, a littlie confusing. 

`This way` means `automatic certificates genreated by skuba` or `replacing the generated ones with custom`. I assume you mean the first one?

Reply:
nice catch.  It'll be changed as
```
Administrators might choose to replace these server's certificates issued by the custom _or_ trusted CA certificate.
This way, the user does not have to add specific certificates to their trusted keychain.
```